src/main/java/com/fasterxml/jackson/annotation/JsonFormat.java

```diff
@@ -518,21 +536,41 @@ public static class Value
          */
         private final Features _features;
 
+        /**
+         * @since 2.21
+         */
+        private final byte _radix;
```

Comment:
let's not be stingy; just use `int` for simplicity (`byte` range is fine of course)